src/crimson/os/seastore/transaction_manager.cc

```diff
@@ -747,6 +747,9 @@ TransactionManager::get_extents_if_live(
               LBAMappingRef& pin) -> Cache::get_extent_iertr::future<>
           {
             auto pin_paddr = pin->get_val();
+            if (pin_paddr.get_addr_type() != paddr_types_t::SEGMENT) {
+              return seastar::now();
```

Comment:
This isn't implemented/tested yet regarding to RBM.

I thought it might become a ZERO pin in the current implementation, but I don't find direct evidence from debug log yet.

Reply:
Ah, yes.